### DISCUSSION TITLE
Keeps redirecting, can't apply

### DISCUSSION THREAD

**Discussion by xzhdev** [community/community#126501] [GitHub Education]
### Hi everyone,

Open the student package certification address: [https://education.github.com/discount requests/application will be automatically redirected to https://github.com/pricing, unable to complete the student certification](https://education.github.com/discount_requests/application%E4%BC%9A%E8%87%AA%E5%8A%A8%E9%87%8D%E5%AE%9A%E5%90%91%E5%88%B0https://github.com/pricing%EF%BC%8C%E6%97%A0%E6%B3%95%E5%AE%8C%E6%88%90%E5%AD%A6%E7%94%9F%E8%AE%A4%E8%AF%81)

**Comment by NicholasStone** [comment]
I have same issue about this. Does apply policy changed?

**Reply by NicholasStone** [reply]
BTW, they send an email and told me my application has been denied due to device camera issue, but I can't even reach this apply page by now.

**Reply by xzhdev** [reply]
They didn't reply to my ticket, I'm speechless.

**Comment by kaktus-1** [comment]
Same problem here. It keeps redirecting. I was already a member of this program and just wanted to reapply. I fulfil all the requirements

**Comment by wovensoul** [comment]
bump, same issue, can someone ping me when there's a fix?

**Comment by avalosnahuel** [comment]
when I try to login to certifications page and click on "login with github" I get this error : 

An unexpected error occurred
Correlation Id: 934db4a0-f2df-4be5-a895-edf52c08bcaf

url is : https://examregistration.github.com/ssovalidation?browser_session_id=c4234c49c5aefbe5779f957de333b892ef72a7e630f62d4ffada1a92b4b8112d&code=a945e5cc775058db012f&state=CfDJ8EwLU6OCPpRHrvtjePfB6eBIAR3IvM9JO0AiCXMSVjTepNdWJXLiaUd1k0ccsfDioTqEAD84xxkbXaG2i7nIxYhEyH6MPTjqFBPNZRzw1IL1qzcGLxwLvbF08zqkIbqNsFiUtms_TTvntAphrZgYUW34oCYjT-eyMEL6cptWE5u-dXnQZKo9haG4weQoqZg1ACkisSk6AdfWWaArg1oTe9COj4Pb9iX4VmLAC9dZpSw5cTeIg5wENC2RtDJjmEFG2A

**Comment by ScoobyDid** [comment]
I think i found the issue; it says that max photo size is 1mb; i have been trying to upload an image that was ~950kb, then i got "413 (Payload Too Large)" error in console; so i reduced the size of image to 671kb and it worked
